lib/metrics.js

```diff
@@ -17,9 +17,6 @@ const metricsIds = {
 
 module.exports = {
   hiddenMetrics: [
-    metricsIds.FV,
-    metricsIds.VC100,
-    metricsIds.VC85,
```

Comment:
Ok, would you agree if a user specifies a metric it should be shown in the graph but if not it should revert to showing the defaults? Any suggestions?